### PR TITLE
Sellsy Contact from Zedental user service

### PR DESCRIPTION
**Pull request by ManyBaby**

### CODE REVIEW

File: app/services/sellsy_contact_from_user.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Finds or create a Sellsy Contact given a ZeDental user
+class SellsyContactFromUser
+  include Sellsy
```
Comment (DavidGeismarLtd) [inline app/services/sellsy_contact_from_user.rb:3]: where does Sellsy come from ?

File: app/services/sellsy_contact_from_user.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Finds or create a Sellsy Contact given a ZeDental user
+class SellsyContactFromUser
```
Comment (DavidGeismarLtd) [inline app/services/sellsy_contact_from_user.rb:2]: SUggestion : namespascing with `module` `SellSy` `class` `ContactFromUserService`

File: lib/sellsy.rb [hunk 3]
```diff
@@ -23,9 +23,10 @@ def send_request(method, params)
 
     # Fetch Request
     res = http.request(req)
-    puts "Response HTTP Status Code: #{res.code}"
-    puts "Response HTTP Response Body: #{res.body}"
+    Rails.logger.info "Response HTTP Status Code: #{res.code}"
```
Comment (DavidGeismarLtd) [inline lib/sellsy.rb:26]: wheres the rest of the file ?

File: app/services/sellsy_contact_from_user.rb (hunk 1 above)
Comment (ManyBaby) [reply app/services/sellsy_contact_from_user.rb:3]: Sellsy is a module in `/lib/sellsy.rb`

File: app/services/sellsy_contact_from_user.rb (hunk 2 above)
Comment (ManyBaby) [reply app/services/sellsy_contact_from_user.rb:2]: makes sense

File: lib/sellsy.rb (hunk 3 above)
Comment (ManyBaby) [reply lib/sellsy.rb:26]: here 🤷🏾‍♂️

Comment (DavidGeismarLtd) [reply lib/sellsy.rb:26]: there's only a very small subsection of the file displayed here

Comment (DavidGeismarLtd) [reply lib/sellsy.rb:26]: checked it out locally are we bound to use Net:HTTP ? wouldnt it be cleaner to use httpparty for instance ?

Comment (ManyBaby) [reply lib/sellsy.rb:26]: I don't know how to do Oauth 1.0 with httparty.

**Pull request merged**